src/main/java/io/github/ashisbored/playerpronouns/PlayerPronouns.java

```diff
@@ -59,6 +93,23 @@ public void onInitialize() {
             }
         });
 
+        ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
+            if(config.enablePronounDBSync()) {
+                String pronounDbUrl = "https://pronoundb.org/api/v1/lookup?platform=minecraft&id=%s".formatted(handler.getPlayer().getUuid());
+                try(CloseableHttpClient client = HttpClients.createMinimal();
```

Comment:
Has this been tested on the dedicated server? Last time I tried using the Apache HTTP client, I believe it was only available on the client

Reply:
ah good point, i tested in dev env but ill build and throw it one one real quick